JWT/JWT.cs

```diff
@@ -173,6 +173,25 @@ private static void Verify(string decodedCrypto, string decodedSignature, string
                     throw new SignatureVerificationException("Token has expired.");
                 }
             }
+            // verify nbf claim https://tools.ietf.org/html/draft-ietf-oauth-json-web-token-32#section-4.1.5
+            if (payloadData.ContainsKey("nbf") && payloadData["nbf"] != null)
```

Comment:
Doing as you suggest would remove symmetry with the 'exp' claim validation behavior. If that was done I would expect both to be changed, and they would lose the "Claim X must be an integer" exception (unless you nested the 'is int' check).